spanner/integration_test.go

```diff
@@ -4632,6 +4663,204 @@ func TestIntegration_GFE_Latency(t *testing.T) {
 	DisableGfeLatencyAndHeaderMissingCountViews()
 }
 
+func TestIntegration_Bit_Reversed_Sequence(t *testing.T) {
+	skipEmulatorTest(t)
+	t.Parallel()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	defer cancel()
+	// create table with bit reverse seq options
+	client, dbPath, cleanup := prepareIntegrationTest(ctx, t, DefaultSessionPoolConfig, statements[testDialect][testTableBitReversedSeqStatements])
+	defer cleanup()
+
+	var tests = []struct {
+		name    string
+		test    func() error
+		wantErr error
+	}{
+		{
+			name: "success: inserted rows should have auto generated keys",
+			test: func() error {
+				var (
+					values [][]interface{}
+					err    error
+				)
+				_, err = client.ReadWriteTransaction(ctx, func(ctx context.Context, tx *ReadWriteTransaction) error {
+					iter := tx.Query(ctx, NewStatement("INSERT INTO T (value) VALUES (100), (200), (300) THEN RETURN id, counter"))
+					values, err = readAllBitReversedSeqTable(iter, true)
+					return err
+				})
+				if len(values) != 3 {
+					return errors.New("expected 3 rows to be inserted")
+				}
+				counter := int64(0)
+				for i := 0; i < 3; i++ {
+					newID, newCounter := values[i][0].(int64), values[i][1].(int64)
+					if newID <= 0 {
+						return errors.New("expected id1, id2, id3 > 0")
+
+					}
+					if newCounter < counter {
+						return errors.New("expected c3 >= c2 >= c1")
+					}
+					counter = newCounter
+				}
+				iter := client.Single().Query(ctx, NewStatement("SELECT GET_INTERNAL_SEQUENCE_STATE('seqT')"))
```

Comment:
can you please check this also?
I think this should throw some error like `No matching signature for function SPANNER:GET_INTERNAL_SEQUENCE_STATE for argument types: STRING `